docs/readmes/bazel/bazel.md

```diff
@@ -0,0 +1,510 @@
+---
+id: bazel_basics
+title: AGW with bazel
+hide_title: true
+---
+
+# AGW with bazel
+
+This page provides a bazel-specific overview on building and testing the Magma Access Gateway (AGW).
+This documentation is aimed mainly at Magma developers. For general bazel support at Magma there is also the Slack channel [#bazel](https://magmacore.slack.com/archives/C033S1CEAUU).
```

Comment:
Does the markdown here support opening links in new tabs, e.g., `[link](url){:target="_blank"}`? Would be nice to use this for external links. Maybe there is even another syntax for external links, in order to render them differently?

Reply:
The curly braces don't work, but HTML does work, e.g. `<a href="http://example.com/" target="_blank">Hello, world!</a>
`. Is that acceptable? The guide https://github.com/magma/magma/wiki/Contributing-Documentation does not mention this.

Reply:
If the linter does not complain, I would say it's acceptable - but this is more or less a NIT, i.e., decide yourself if this is worth the clunky syntax.

Reply:
I will keep this point open to decide at a later point.

Reply:
Closing this comment: I think using HTML here would differ too much from the rest of the docs and might make it harder to maintain in the future.